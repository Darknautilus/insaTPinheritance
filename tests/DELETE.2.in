C circle 1 2 3
L line 1 1 2 2 
PL poly 1 1 2 2 3 3 4 4
OA agr circle line poly
DELETE circle line
LIST
DELETE poly
LIST
EXIT
