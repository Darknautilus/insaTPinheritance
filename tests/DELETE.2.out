OK
OK
OK
OK
OK
OA agr poly
PL poly 1 1 2 2 3 3 4 4
OK
OA agr
